data/dialog.yaml

```
@@ -286,3 +286,101 @@ Wendy Escort Complete:
   actions:
     use drink:
     - Whisky
+
+Escort Gerud Beer Covet Offer:
+  conversation:
+    - label 1:
+      - print(You decide to rest a little at the Los Gerudos Grand Hostel, and have a glass of Wine; you start drinking it. A few minutes later, a man sits next to you, at your left. He starts to talk:)
+      - print(“- Hey. Have I seen you at the Los Gerudos Prison in the sheriff's office? says the man)
+      - print(- Maybe, once I went there because I wanted to help him for some kind of missions; you respond to the man.)
```

```suggestion
      - print(- Maybe, once I went there because I wanted to help him for some kind of missions; you respond.)
```
This flows better I think.